Source/FakeItEasy.Tests/ATests.cs

```diff
@@ -79,17 +66,17 @@ public void CollectionOfFakes_should_delegate_to_fake_creator()
             A.CallTo(() => creator.CollectionOfFake<IFoo>(10)).Returns(returnedFromCreator);
 
             // Act
-            var result = A.CollectionOfFake<IFoo>(10);
+            object result = A.CollectionOfFake<IFoo>(10);

```

Comment:
FluentAssertions wouldn't give me .BeSameAs() so long as it knew that result was a collection. It was this or an ugly cast on (new) line 72.